packages/@aws-cdk/aws-iam/lib/role.ts

```diff
@@ -140,3 +166,13 @@ function createAssumeRolePolicy(principal: PolicyPrincipal) {
             .addPrincipal(principal)
             .addAction(principal.assumeRoleAction));
 }
+
+function validateMaxSessionDuration(duration?: number) {
```

Comment:
Does not need to be able to be `undefined`.

Reply:
I will remove the `if` from the call site